js/africa_map.js

```diff
@@ -42,10 +42,12 @@ let southWest = L.latLng(53.85252660044951, 107.75390625),
   bounds = L.latLngBounds(southWest, northEast);
 
 let map = L.map('map', {
-  maxBounds: bounds,
+  zoomControl: true,
```

Comment:
@Ugerpaul17 , this puts the zoom control for all three maps. We'll more likely need it for regional